packages/by-components/src/icons/arrows.rs

```diff
@@ -83,6 +83,7 @@ pub fn ChevronUp(
     #[props(default = "none".to_string())] fill: String,
     #[props(default = "24".to_string())] height: String,
     #[props(default = "".to_string())] class: String,
+    #[props(default = "".to_string())] color: String,
```

Comment:
```suggestion
    #[props(default = "black".to_string())] color: String,
```
공용 컴포넌트기 때문에, 하위호환성이 보장안되면 다른 서비스의 UI 에 영향을 줄 수 있습니다.